pkg/codegen/java/gen_test.go

```diff
@@ -35,6 +35,10 @@ var javaSpecificTests = []*test.SDKTest{
 		Directory:   "mini-awsx",
 		Description: "Regression tests extracted from trying to codegen awsx",
 	},
+	{
```

Comment:
Hmmm, this one is a bit surprising to me. My mental model was that we borrow tests from the common test suite and then add Java-specific regression tests here. Like `mini-awsnative` is Java specific test. But it would seem that `output-funcs-edgeorder` should be coming from pulumi/pulumi? It's not there?

Reply:
That is I was expecting it to be in test.PulumiPulumiSDKTests?

Reply:
I'll take care of sorting this out later, not blocking.